src/utils/platform.ts

```diff
@@ -4,14 +4,33 @@ const platform = typeof navigator === 'undefined' ? 'some platform' : navigator.
 export const isChromium = userAgent.includes('chrome') || userAgent.includes('chromium');
 export const isThunderbird = userAgent.includes('thunderbird');
 export const isFirefox = userAgent.includes('firefox') || isThunderbird;
-export const isVivaldi = userAgent.includes('vivaldi');
+// Vivaldi attempts to hide itself from User Agent sniffing so we use the following facts:
+//  1. Vivaldi does not support navigator.userAgentData at all (it is undefined)
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/135728331-9ae58a90-0706-4fbb-8bbe-e8e169a1c969.png)
I assume this is only true for non-devtools context?

Reply:
I looked into it and in extension contexts `userAgentData` is supported, but does not contain Vivaldi name.